src/app/components/TrainingsPage/TrainingsPage.scss

```diff
@@ -0,0 +1,110 @@
+@import '/src/styles/mixins.scss';
+
+.trainings {
+
+    &__title {
+        color: var(--Black, #191919);
```

Comment:
Weź kolor ze zmiennych.

Reply:
Te nie wiem czemue zmienne z css sa uzywane zamiast zmiennych scss